apps/sms/views/conversation/js/conversation.js

```diff
@@ -739,7 +773,7 @@ var ConversationView = {
     }
   },
 
-  beforeEnterComposer: function conv_beforeEnterComposer(args) {
+  beforeEnterComposer: function conv_beforeEnterComposer() {
```

Comment:
if you change this line, then just use the ES6 shortcut and remove the "function" part